std/experimental/xml/cursor.d

```diff
@@ -0,0 +1,989 @@
+/*
+*             Copyright Lodovico Giaretta 2016 - .
+*  Distributed under the Boost Software License, Version 1.0.
+*      (See accompanying file LICENSE_1_0.txt or copy at
+*            http://www.boost.org/LICENSE_1_0.txt)
+*/
```

Comment:
While I understand your position, this is what many Phobos modules do (json, random, traits, uri,, ...). The reason is that one is the "normative" Boost header that rules the file, while the other is just a directive to print those informations in the docs.
The positioning of the ddoc license section and its formatting make it less evident than this other comment and so less useful to one that reads the source, while this other comment is also what is recommended by the Boost website.